package.json

```diff
@@ -1,27 +1,57 @@
 {
-  "name": "xnft-quickstart",
-  "version": "2.0.0",
-  "description": "",
-  "main": "index.js",
-  "author": "",
-  "license": "ISC",
+  "main": "node_modules/expo/AppEntry.js",
```

Comment:
are you sure you can get away without registerRootComponent in an index.js file? that's the first set of instructions under: https://docs.expo.dev/workflow/web/

Reply:
Yea I added registerRootComponent to `App.tsx` to remove some magic. And I'm using app.json define it as entrypoint.

Reply:
My goal was that `src` contains all App code and ppl can mostly ignore everything outside it. 
Only exceptions are `template.html` and `assets/`